.nix/arion-xcvm.nix

```diff
@@ -0,0 +1,70 @@
+{ pkgs, packages, ... }:
+pkgs.arion.build {
+  modules = [
+    ({ pkgs, ... }:
+      let
+        default-db = {
+          name = "xcvm";
+          host = "127.0.0.1";
+          user = "xcvm";
+          password = "xcvm";
+          port = 1337;
+        };
+        juno-db = default-db // {
+          name = "juno";
+        };
+        squid-db = default-db // {
+          name = "squid";
+        };
+      in {
+        config.project.name = "Composable Finance XCVM devnet";
+        config.services = {
+
+          # ============ COMMON ===============
+          db = import ./services/postgres.nix {
+            database = default-db;
+            version = "14";
+            init-scripts = pkgs.writeTextFile {
+              name = "init";
+              text = ''
+                CREATE DATABASE juno WITH OWNER xcvm;
```

Comment:
Don't know the nix language exactly but can we change these owner names to "database.user". Eg in rust:
```rust
format!("... OWNER {}", database.user);
```

Reply:
done